api/src/main/java/io/minio/MinioClient.java

```diff
@@ -3818,6 +3818,10 @@ public void enableVersioning(String bucketName)
   /**
    * Enables object versioning feature in a bucket.
    *
+   * Versioning feature is available only for erasure coded and distributed erasure coded setups
```

Comment:
`minio-java` is meant to any S3 service not limiting to MinIO server. This doc is not relevant here.